### PR TITLE
Polish

### PR DESCRIPTION
**Pull request by izeye**
This PR fixes some typos and polishes trivial stuff.

### CODE REVIEW

File: spring-boot-project/spring-boot/src/test/java/org/springframework/boot/web/embedded/tomcat/TomcatReactiveWebServerFactoryTests.java [hunk 1]
```diff
@@ -59,13 +59,14 @@ protected TomcatReactiveWebServerFactory getFactory() {
...
+		TomcatContextCustomizer[] customizers = new TomcatContextCustomizer[4];
+		Arrays.setAll(customizers, (i) -> mock(TomcatContextCustomizer.class));
+		factory.setTomcatContextCustomizers(
+				Arrays.asList(customizers[0], customizers[1]));
+		factory.addContextCustomizers(customizers[2], customizers[3]);
 		this.webServer = factory.getWebServer(mock(HttpHandler.class));
-		InOrder ordered = inOrder((Object[]) listeners);
-		for (TomcatContextCustomizer listener : listeners) {
+		InOrder ordered = inOrder((Object[]) customizers);
+		for (TomcatContextCustomizer listener : customizers) {
```
Comment (snicoll) [inline spring-boot-project/spring-boot/src/test/java/org/springframework/boot/web/embedded/tomcat/TomcatReactiveWebServerFactoryTests.java]: Can you please rename `listener` to `customizer` there as well?

File: spring-boot-project/spring-boot/src/test/java/org/springframework/boot/web/embedded/tomcat/TomcatReactiveWebServerFactoryTests.java [hunk 2]
```diff
@@ -145,13 +146,14 @@ public void addNullProtocolHandlerCustomizersShouldThrowException() {
...
+		TomcatConnectorCustomizer[] customizers = new TomcatConnectorCustomizer[4];
+		Arrays.setAll(customizers, (i) -> mock(TomcatConnectorCustomizer.class));
+		factory.setTomcatConnectorCustomizers(
+				Arrays.asList(customizers[0], customizers[1]));
+		factory.addConnectorCustomizers(customizers[2], customizers[3]);
 		this.webServer = factory.getWebServer(handler);
-		InOrder ordered = inOrder((Object[]) listeners);
-		for (TomcatConnectorCustomizer listener : listeners) {
+		InOrder ordered = inOrder((Object[]) customizers);
+		for (TomcatConnectorCustomizer listener : customizers) {
```
Comment (snicoll) [inline spring-boot-project/spring-boot/src/test/java/org/springframework/boot/web/embedded/tomcat/TomcatReactiveWebServerFactoryTests.java]: Ditto

File: spring-boot-project/spring-boot/src/test/java/org/springframework/boot/web/embedded/tomcat/TomcatServletWebServerFactoryTests.java [hunk 3]
```diff
@@ -162,13 +162,14 @@ public void tomcatListeners() {
...
+		TomcatContextCustomizer[] customizers = new TomcatContextCustomizer[4];
+		Arrays.setAll(customizers, (i) -> mock(TomcatContextCustomizer.class));
+		factory.setTomcatContextCustomizers(
+				Arrays.asList(customizers[0], customizers[1]));
+		factory.addContextCustomizers(customizers[2], customizers[3]);
 		this.webServer = factory.getWebServer();
-		InOrder ordered = inOrder((Object[]) listeners);
-		for (TomcatContextCustomizer listener : listeners) {
+		InOrder ordered = inOrder((Object[]) customizers);
+		for (TomcatContextCustomizer listener : customizers) {
```
Comment (snicoll) [inline spring-boot-project/spring-boot/src/test/java/org/springframework/boot/web/embedded/tomcat/TomcatServletWebServerFactoryTests.java]: Ditto

File: spring-boot-project/spring-boot/src/test/java/org/springframework/boot/web/embedded/tomcat/TomcatReactiveWebServerFactoryTests.java (hunk 1 above)
Comment (izeye) [reply spring-boot-project/spring-boot/src/test/java/org/springframework/boot/web/embedded/tomcat/TomcatReactiveWebServerFactoryTests.java]: @snicoll Oops, I missed them somehow 😅 Thanks for catching them 👍 I've updated and force-pushed.

**Comment by snicoll** [comment]
Thanks again @izeye!

**Pull request closed without merging**